### PR TITLE
Update Place Tomba Dougoula

### PR DESCRIPTION
**Pull request by nvkelso**
Updating `data/126/021/452/3/1260214523.geojson` using the [WOF Editor](https://github.com/iandees/wof-editor)

### CODE REVIEW

**Comment by nvkelso** [comment]
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/2172

**Review by 5212BABA — APPROVED** [review]
Thank you very much for validating my suggestion by correcting erroneous names

File: data/126/021/452/3/1260214523.geojson [hunk 1]
```diff
@@ -28,18 +28,24 @@
     "lbl:max_zoom":18.0,
     "lbl:min_zoom":13.5,
     "mz:hierarchy_label":1,
-    "mz:is_current":-1,
+    "mz:is_current":1,
     "mz:min_zoom":12.0,
+    "name:eng_x_preferred":[
+        "Tomba Kanssa"
+    ],    "name:eng_x_variant":[
```
Comment (stepps00) [inline data/126/021/452/3/1260214523.geojson:35]: Hmm.. looks like invalid geojson? I also see an extra return char at the tail end of this record. If you're able to Exportify the record, this should correct itself..

**Review by stepps00 — CHANGES_REQUESTED** [review]
See Exportify coment

**Comment by nvkelso** [comment]
Replaced with https://github.com/whosonfirst-data/whosonfirst-data-admin-gn/pull/41.

Back story here is I missed setting the English preferred name in the tool so edited that manually in GH.com interface, which then garbled up the formatting. Always better to use Write Field to recreate the work – or to pull it into local checkout and apply exportify manually.

**Pull request closed without merging**